akka-sample-distributed-workers-scala/worker.md

```diff
@@ -0,0 +1,19 @@
+# The Worker Nodes
+
+`Worker` actors and the `Master` actor interact as follows:
+
+1. `Worker` actors register with the `Master` so the master knows they are available and ready to take on work.
```

Comment:
They register with the receptionist, the `Master` subscribes to the receptionist.